outputs.tf

```diff
@@ -0,0 +1,8 @@
+output "webapp_url" {
+ value = "https://${azurerm_app_service.main.default_site_hostname}"
+}
+
+output "webhook_url" {
+  value = "https://${azurerm_app_service.main.site_credential[0].username}:${azurerm_app_service.main.site_credential[0].password}@${azurerm_app_service.main.name}.scm.azurewebsites.net/docker/hook"
+  #sensitive   = true
```

Comment:
Why is this line commented out? You **do** want `sensitive = true` for this. It contains your deployment password, and you're currently leaking it in your Travis logs 

Expand the "Deploying application" section at the bottom of the logs:
https://app.travis-ci.com/github/ArkSurfer/DevOps-Course-Starter/builds/234971711